plugins/reporters/spdx/src/funTest/assets/spdx-document-reporter-expected-output.spdx.json

```diff
@@ -4,7 +4,7 @@
   "creationInfo" : {
```

Comment:
commit-msg: 
- sepc -> spec
- rely -> relies

Reply:
Instead of "rely -> relies" I'm now using plural "tools".